client/api_bucket.go

```diff
@@ -159,17 +99,30 @@ func (c *Client) CreateBucket(ctx context.Context, bucketName string, primaryAdd
 	if err != nil {
 		return "", err
 	}
-	signedMsg, err := c.GetCreateBucketApproval(ctx, createBucketMsg)
+
+	accAddress, err := sdk.AccAddressFromHexUnsafe(primaryAddr)
+	if err != nil {
+		return "", err
+	}
+
+	sp, err := c.GetStorageProviderInfo(ctx, accAddress)
 	if err != nil {
 		return "", err
 	}
 
+	families, err := c.QuerySpAvailableGlobalVirtualGroupFamilies(ctx, sp.Id)
```

Comment:
I think we'd better use the optimal API to get one VGF here?

Reply:
fixed